AUTHORS.rst

```diff
@@ -0,0 +1,6 @@
+
+Authors
+=======
+
+* Chris Gemignani - www.github.com/juiceinc/recipe
```

Comment:
Generous of you to give me authorship on this one. Eventually I'll earn it.  :-P

Reply:
so approved?